src/umessage.rs

```diff
@@ -11,26 +11,6 @@
  * SPDX-License-Identifier: Apache-2.0
  ********************************************************************************/
 
-#[derive(Debug)]
-pub struct SerializationError {
-    message: String,
-}
+mod utransport;
```

Comment:
Finding `UTransport` in module `umessage::utransport` feels very strange to me. FMPOV the `utransport` module should be at the same level as `umessage`.

Reply:
Looks like I missed a few things, also around the cloudevent module. Working on it...

Reply:
Ok; I "repaired" the so-far-missing transport and cloudevent modules, hopefully didn't forget anything this time. Also I took this as an opportunity to go through the entire codebase, unifying and cleaning up ```use``` statements everywhere. Update the crate comments as well, and pulled in a little ./tools script that runs cargo fmt, clippy and doc locally for convenience.